### DISCUSSION TITLE
Waroomへの再招待機能の追加

### DISCUSSION THREAD

**Discussion by luup-saniwa** [topotal/waroom-roadmap#66] [Feature Request]
## Why
人によって、Waroomのログインについてすぐには確認や作業時間が取れないことがあるため、招待リンクの期限が切れてしまうことがあるため、再招待機能の追加実装をお願いしたいです

## What
Waroomへの再招待機能の追加を実装する

**Comment by luup-saniwa** [comment]
なお、現在はパスワードリセットによって対象者のログインが可能となっているため、緊急ではありません。

**Reply by sawa-zen** [reply]
@luup-saniwa 
ありがとうございます！現在同じメールアドレスに招待メールを送るとエラーになってしまうのも確認できたので、再招待できるように修正いたします 🙏🏻

**Reply by sawa-zen** [reply]
@luup-saniwa 
未ログインの招待済みのアカウントに再招待をした際にエラーになるバグを修正いたしました！招待リンクの期限にかかわらず同じメールアドレスに対して招待メールが送れるようになったかと思います。

こちらの discussion は一度クローズしますが、もしまた問題がございましたらコメントいただけますと助かります 🙏🏻
